src/Store.lua

```diff
@@ -142,7 +133,7 @@ function Store:dispatch(action)
 	if action.type == nil then
 		error("Actions may not have an undefined 'type' property. " ..
 			"Have you misspelled a constant? \n" ..
-			inspect(action), 2)
+			tostring(action), 2)
```

Comment:
is tostring() going to give actiomable information when it’s a table with the wrong format?

Reply:
Maybe not! But this is the kind of error that we're much more likely to catch in dev (it was an error in 1.x before). It didn't seem like it justified all of the `inspect` logic, some of which actually had to be adjusted to work in non-Luau contexts for the sake of this repo's CI

Reply:
Is it possible for us to safely get action.name here? That would be more useful than the pointer/id.

Reply:
Not sure I follow; `action.name` doesn't have any particular meaning. The only required structure for an action is that it's a table with a `type` field (maybe that's what you're thinking of?). This error will only be thrown in the event that an action without a `type` is dispatched.

I think the way this error happens in practice is if you accidentally dispatch something other than a proper action in your `mapDispatchToProps`, which would make whatever you were trying to do fail altogether. Hence why I suggested that this is likely an error that folks find and resolve during development.

Reply:
Sorry, `action.type` is the correct nomenclature. In the Lua App codebase we have a "name" argument (ultimately Script.name) that gets assigned to "type" in Action.lua. Makes things a little confusing.

I'm just thinking that it will provide a nicer error message and doesn't require you to go through the expense of visiting the whole table. Not a big deal.